composer.json

```diff
@@ -11,12 +11,11 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "php": ">=5.6,<8.0-DEV"
+        "php": ">=7.4"
     },
     "require-dev": {
-        "composer/composer": "~1.1",
-        "phpunit/phpunit": "~5.3",
-        "etsy/phpunit-extensions": "0.7.*"
+        "composer/composer": "^2",
```

Comment:
what's this for?

Reply:
ah this is mostly for step debugging, see purple box here: https://getcomposer.org/doc/articles/plugins.md#plugin-package

Reply:
are you sure you're not after
`"composer-plugin-api": "^1.0.0 || ^2.0.0"`
or just v2?

Reply:
yeah need to add this, the composer thing is for debugging, so want to keep as dev requirement.